assets/SimpliGov starter email template.html

```diff
@@ -0,0 +1,199 @@
+  <!-- Default styling of text is applied to the wrapper div. Be sure to use text that is large enough and has a high contrast with the background color for people with visual impairments. -->
+  <div role="article" aria-label="An email from the MBTA" lang="en" style="
+        background-color: #fafafa !important;
+        color: #2b2b2b;
+        font-family: 'Helvetica Neue', Helvetica, Arial, Verdana, sans-serif;
+        font-size: 18px;
+        font-weight: 400;
+        line-height: 28px;
+        margin: 0 auto;
+        max-width: 600px;
+        padding: 40px 20px 40px 20px;
+      ">
+    <!-- Logo section and header -->
+    <header style="padding-bottom: 16px;">
+      <center>
+        <img src="https://mbta.prod.simpligov.com/prod/portal/file/51d7628117ce4c0c9cb0000188390b62.png?t=1633355953036"
+          alt="MBTA logo" width="225" />
+      </center>
+    </header>
+
+    <!-- Main content -->
+    <main>
+      <!-- This div is purely presentational, providing a container for the message. -->
+      <div style="background-color: #fff; border-radius: 4px; padding: 24px 24px">
+        <!-- This ghost table is used solely for padding in Word-based Outlook clients. -->
+        <!--[if (gte mso 9)|(IE)]>
+                <table cellspacing="0" cellpadding="0" border="0" width="600" align="center" role="presentation"><tr><td style="background-color: #fff;font-family: 'Avenir Next', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; padding: 24px 48px 24px 48px;">
+                <![endif]-->
+
+        <!-- Body copy -->
+        <h1>
+          We've received your application!
+        </h1>
+        <p>
+          We're verifying your eligibility for the
+          <a href="https://www.mbta.com/fares/reduced/senior-charliecard">Senior CharlieCard Program</a>.
+        </p>
+
+        <p>
+          We'll send you an email in 7-10 business days if your application is
+          approved or denied, or if you need to submit more information. If
+          your application is approved, you'll receive your Senior CharlieCard
+          in the mail.
+        </p>
+
+        <h2>Questions?</h2>
+        <p>
+          Contact
+          <a href="https://www.mbta.com/customer-support">Customer Support using their online form</a>
+          or call the numbers below.
+        </p>
+        <p>
+          Monday &ndash; Friday: 6:30 AM &ndash; 8 PM<br />
+          Saturday &ndash; Sunday: 8 AM &ndash; 4 PM
+        </p>
+        <p>
+          <strong>Main Hotline:</strong>
+          <a href="tel:617-222-3200">617-222-3200</a><br />
+          Toll Free: <a href="tel:800-392-6100">800-392-6100</a><br />
+          TTY: <a href="tel:617-222-5146">617-222-5146</a>
+        </p>
+
+        <h2>Tell us about your experience</h2>
+        <p>Take our two-minute survey. Your feedback helps us improve the Senior CharlieCard application.</p>
```

Comment:
**suggestion:** I would suggest either deleting this line or commenting it out until we have a link to the survey. If it went out to riders this way it seems like it would be pretty confusing.

Reply:
@arkadyan the senior Qualtrics survey is live and linked in the button/table below. Are changes needed here?